src/components/cards/CohortGroup.svelte

```diff
@@ -1,34 +1,51 @@
 <script>
-    import CohortCard from './CohortCard.svelte';
+  import CohortCard from './CohortCard.svelte'
 
-    export let entries = [];
+  export let entries = []
 
-    // Since 'entries' is a prop, it will be passed from the parent component
-    // where this component is used.
+  // Since 'entries' is a prop, it will be passed from the parent component
+  // where this component is used.
 </script>
 
-<div class="h-screen snap-start pt-40 flex bg-slate-200 bg-slate-800 text-white">
+<div class="py-10 text-white bg-[#f0f0f0]">
   <!-- Left Column -->
-  <div class="w-1/2 pt-5">
-    <p class="text-2xl pt-5 text-center">Ready to Apply?</p>
-    <div class="max-w-5xl mx-auto mt-5 text-center">
-      <h1 class="block text-5xl font-bold md:text-5xl lg:text-6xl">
-        JOIN THE COHORT THAT'S RIGHT FOR YOU.
-      </h1>
+  <div class="flex">
+    <div class="w-full h-fit">
+      <p
+        class="text-3xl md:text-4xl lg:text-5xl font-bold pt-5 text-center text-black"
+      >
+        READY TO APPLY?
+      </p>
+      <div class="max-w-5xl mx-auto mt-5 text-center">
+        <h1
+          class="block text-2xl font-semibold md:text-3xl lg:text-4xl text-black"
+        >
+          JOIN THE COHORT THAT'S RIGHT FOR YOU.
+        </h1>
+      </div>
+    </div>
+  </div>
+  <div class="flex py-20">
+    <div class="flex flex-col lg:flex-row w-full items-center justify-center gap-16">
+      <CohortCard />
```

Comment:
Instead of putting in two blank cohort cards, we should put in all the cohorts like was done in the commented out section. This might need to be done as part of a different PR, as it will require updating the roles pages to actually show anything here.

Reply:
Fixed in latest commit.